.github/CODEOWNERS

```
@@ -1,9 +1,13 @@
-# This CODEOWNERS file sets the individuals responsible for code in the sdk-docs repository.
+# This CODEOWNERS file sets the individuals responsible for code in the sdk-docs repository
 
-# These users are the default owners for everything in the repo.
-# They will be requested for review when someone opens a pull request.
+# These users are the default owners for everything in the repo
+# They will be requested for review when someone opens a pull request
 
-# You can also specify code owners for specific directories or files.
-# For example:
+* @zksync-sdk/devxp
```

Should we expand this to also include `txfusion`?
```suggestion
* @zksync-sdk/devxp @zksync-sdk/txfusion 
```

---

Wasn't sure if they owned all of the docs as well to require them as part of the reviewers for `*`.

---

Ah, based on the other comment, limiting docs/repo structure changes can go through `devxp` for now